### PR TITLE
Add TTL 74138

### PR DESCRIPTION
**Pull request by btb**
split off from #1461

### CODE REVIEW

**Review by maehne — DISMISSED** [review]
Thanks for the PR @btb! Looks good to me!

**Review by MarcinOrlowski — CHANGES_REQUESTED** [review]
Please address linter complaints.

File: src/main/resources/resources/logisim/strings/std/std_fr.properties [hunk 1]
```diff
@@ -768,7 +768,8 @@ TTL7411 = 7411 : triple 3 entrées ET porte d'entrée
 TTL74125 = 74125 : tampon quadruple bus, sorties à trois états, validation négative
 TTL74157 = 74157 : sélecteur de données quadruple de 2 lignes à 1 ligne
 TTL74158 = 74158 : quadruple 2-lignes to 1 ligne sélecteur, Sorties inversées
```
Comment (maehne) [inline src/main/resources/resources/logisim/strings/std/std_fr.properties:770]: "quadruple 2-lignes to 1 ligne sélecteur, Sorties inversées" -> "sélecteur quadruple de 2 lignes à 1 ligne, sorties inversées"

File: src/main/resources/resources/logisim/strings/std/std_fr.properties [hunk 2]
```diff
@@ -768,7 +768,8 @@ TTL7411 = 7411 : triple 3 entrées ET porte d'entrée
...
-TTL74139 = 74139 : quadruple 2-lignes to 4-lignes décodeur
+TTL74138 = 74138 : 3-lignes to 8-lignes décodeur
```
Comment (maehne) [inline src/main/resources/resources/logisim/strings/std/std_fr.properties:771]: "3-lignes to 8-lignes décodeur" -> "décodeur de 3 lignes à 8 lignes"

File: src/main/resources/resources/logisim/strings/std/std_fr.properties [hunk 3]
```diff
@@ -768,7 +768,8 @@ TTL7411 = 7411 : triple 3 entrées ET porte d'entrée
...
+TTL74139 = 74139 : double 2-lignes to 4-lignes décodeur
```
Comment (maehne) [inline src/main/resources/resources/logisim/strings/std/std_fr.properties:772]: "double 2-lignes to 4-lignes décodeur" -> "décodeur double de 2 lignes à 4 lignes"

**Review by maehne — DISMISSED** [review]
Looks mostly good to me except for the French translation strings.

**Comment by btb** [comment]
> Please address linter complaints.

Having trouble running checkstyle locally:
```
bradleyb@tmbp:logisim-evolution$ ./gradlew checkstyleMain
> Task :checkstyleMain FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':checkstyleMain'.
> Expected file collection to contain exactly one file, however, it contains no files.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
5 actionable tasks: 1 executed, 4 up-to-date
```

Web searches seem to imply there is an issue with gradle plugin dependencies

File: src/main/resources/resources/logisim/strings/std/std_fr.properties (hunk 1 above)
Comment (btb) [reply src/main/resources/resources/logisim/strings/std/std_fr.properties]: ma grammaire est pas bon, mais j'ai essayé

**Comment by btb** [comment]
The linter complaints generated by the previous github workflow have been fixed.

File: src/main/resources/resources/logisim/strings/std/std_de.properties [hunk 4]
```diff
@@ -768,6 +768,7 @@ TTL7411 = 7411: Dreifach 3-Eingangs-UND-Gatter
 TTL74125 = 74125: Vierfach Bus-Puffer, Drei-Status-Ausgänge, negative Freigabe
 TTL74157 = 74157: Vierfach 2 zu 1 Leitungsauswahl
 TTL74158 = 74158: Vierfach 2 zu 1 Leitungsauswahl, negierter Ausgang
+TTL74138 = 74138: 3 zu 8 Decoder
```
Comment (MarcinOrlowski) [inline src/main/resources/resources/logisim/strings/std/std_de.properties:771]: isn't "line" thing important here?

File: src/main/resources/resources/logisim/strings/std/std_pl.properties [hunk 5]
```diff
@@ -768,6 +768,7 @@ TTL7411 = 7411: potrójne 3 wejścia i bramka
 TTL74125 = 74125: bufor czterobusowy, wyjścia trójstanowe, ujemne zezwolenie
 # ==> TTL74157 =
 # ==> TTL74158 =
+# ==> TTL74138 =
```
Comment (MarcinOrlowski) [inline src/main/resources/resources/logisim/strings/std/std_pl.properties:771]:
```ini
TTL74138 = Dekoder 3 linie do 8 linii
```

**Review by MarcinOrlowski — CHANGES_REQUESTED** [review]

File: src/main/resources/resources/logisim/strings/std/std_de.properties (hunk 4 above)
Comment (btb) [reply src/main/resources/resources/logisim/strings/std/std_de.properties:771]: I would think so, but I was following the lead of all the surrounding translations that don't use it. (157,158,139).

I should probably just leave it empty, right?

File: src/main/resources/resources/logisim/strings/std/std_fr.properties [hunk 6]
```diff
@@ -787,7 +788,7 @@ TTL74273 = 74273 : bascule octale en D avec bascule transparente
 TTL74283 = 74283 : Additionneur complet binaire 4 bits
 TTL7430 = 7430 : porte NAND simple à 8 entrées
 TTL7432 = 7432 : quad 2 entrées OU porte OU
-TTL7434 = 74157 : quadruple 2-lignes to 1 ligne sélecteur
+TTL7434 = 7434 : tampon hexagonal
```
Comment (maehne) [inline src/main/resources/resources/logisim/strings/std/std_fr.properties:791]: According to [Wikipedia](https://fr.wikipedia.org/wiki/Liste_des_circuits_intégrés_de_la_série_7400), I suggest to use "six tampons non-inverseurs".

**Review by maehne — CHANGES_REQUESTED** [review]
Except for the Frech translation string for 7434, this PR looks good to me.

**Review by maehne — APPROVED** [review]
Thanks for addressing quickly my review comment!

**Review by maehne — APPROVED** [review]
@MarcinOrlowski: What is open from your side to get this PR merged?

**Review by MarcinOrlowski — APPROVED** [review]

**Review by MarcinOrlowski — CHANGES_REQUESTED** [review]
Oh, I forgot. Please add change log entry to `CHANGELOG` file.

**Comment by BFH-ktt1** [comment]
@MarcinOrlowski : there is an entry in the changelog, could you deblock?

**Pull request merged**